byterun/caml/osdeps.h

```diff
@@ -125,6 +125,15 @@ extern char* caml_stat_strdup_of_utf16(const wchar_t *s);
 */
 extern value caml_copy_string_of_utf16(const wchar_t *s);
 
+/* [caml_utf8_of_utf16(s)] returns a NULL-terminated copy of [s], re-encoded
+   in UTF-8 if [caml_windows_unicode_runtime_enabled] is non-zero or
+   the current Windows code page otherwise.
+
+   The returned string points to a fixed-length, statically-allocated buffer so
+   make sure to copy it if it needs to be preserved.  Also, due to the fixed
+   length of the buffer, the result may be truncated. */
+extern char * win_utf8_string_of_utf16(const wchar_t *s);
```

Comment:
The function name in the documentation comment does not match the exposed name.

Reply:
The function no longer matches the semantics of the comment, either!